x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/data/X-BlockBuilder.java.st

```diff
@@ -57,6 +62,11 @@ $endif$
         return this;
     }
 
+    @Override
+    protected int elementSize() {
+        return $if(BytesRef)$-1$else$$if(boolean)$Byte$elseif(int)$Integer$elseif(long)$Long$else$Double$endif$.BYTES$endif$;
```

Comment:
I think we might already have a constant for this in the build.gradle file. Or we should add one.

Reply:
Ah ha! yes. Fixed, see [00e708c](https://github.com/elastic/elasticsearch/pull/99657/commits/00e708cdc1b4a51d139c5e7cfdf2caf7018dc050)